hg-fast-export.py

```diff
@@ -449,7 +453,7 @@ def verify_heads(ui,repo,cache,force,branchesmap):
 
   # verify that branch has exactly one head
   t={}
-  for h in repo.heads():
+  for h in repo.filtered('visible').heads():
```

Comment:
Will this and the `hidden()` checks above also filter out closed branches which has previously required using the `--force` option? (not a critique of this patch, just curious)

Reply:
If I'm reading the mercurial code right then no, but revisions do have a closesbranch() property so it should be straightforward to detect those from the heads.